python-frontend/typeshed_serializer/runners/tox_runner.py

```diff
@@ -127,8 +127,11 @@ def main(skip_tests=False):
     logger.info(f"Current checksum {current_sources_checksum}")
     logger.info(f"Checksum is computed over {len(source_files)} files")
     if previous_sources_checksum != current_sources_checksum:
-        logger.info("STARTING TYPESHED SERIALIZATION")
-        subprocess.run(["tox"], check=True)
+        if not fail_fast:
+            logger.info("STARTING TYPESHED SERIALIZATION")
+            subprocess.run(["tox"], check=True)
+        else:
+            raise RuntimeError('INCONSISTENT BINARY CHECKSUMS')
```

Comment:
I would have swapped the conditions and the branches to have `if fail_fast: ...`